charts/kserve-resources/templates/servingruntimes.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: serving.kserve.io/v1alpha1
-kind: ServingRuntime
+kind: ClusterServingRuntime
```

Comment:
What happens when this is changed from `ServingRuntime` to `ClusterServingRuntime` during upgrade?

Reply:
@rafvasq did a test on the migration scenario yesterday. Paraphrasing a bit:

> deployed 0.9 and an `isvc`, then upgraded to `0.10`. The `isvc` is untouched (relevant pods redeployed as expected), while both `clusterservingruntimes` and `servingruntimes` are now available. The `servingruntimes` from `v0.9` weren't deleted. The `servingruntimes` still exist (along with the `clusterservingruntimes`)
> then did manually delete the `servingruntimes` to see what happened --> no changes to any pods or `isvc`s


Thanks Rafael!